src/geofabrics/processor.py

```diff
@@ -301,15 +306,17 @@ def run(self):
             self.catchment_lidar.load_tile(lidar_file_path)
 
             # update the dense DEM with a patch created from the LiDAR tile
-            self.dense_dem.add_tile(self.catchment_lidar.tile_array, window_size, idw_power, radius)
+            self.dense_dem.add_tile(tile_points=self.catchment_lidar.tile_array,
+                                    tile_extent=self.catchment_lidar.tile_extent,
+                                    window_size=window_size, idw_power=idw_power, radius=radius)
             del self.catchment_lidar.tile_array
 
         # Filter the LiDAR extents based on the area_to_drop
-        self.catchment_lidar.filter_lidar_extents_for_holes()
+        self.dense_dem.filter_lidar_extents_for_holes()
```

Comment:
Can you clarify in comment what is a "dense_dem"?

Reply:
Good idea. Will do :)

Reply:
I've added a description of the dense_dem object and all others (catchment_lidar, etc) in the `GeoFabricsGenerator` doc-string.